modules/storage/storage-account/main.bicep

```diff
@@ -55,6 +151,30 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
   }
 }
 
+resource blobService 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01' = if (enableBlobService) {
+  name: 'default'
+  parent: storageAccount
+  properties: blobProperties
+}
+
+resource container 'Microsoft.Storage/storageAccounts/blobServices/containers@2022-09-01' = if (enableBlobService) {
```

Comment:
That was an amazing suggestion! I didn't know you can nest resources like that instead of linking them with 'parent' parameter.
Updated in https://github.com/Azure/bicep-registry-modules/commit/06b88359be80e90b6d23b5abd98746a7205b69e8